testsuite/MDAnalysisTests/analysis/test_psa.py

```diff
@@ -50,11 +53,15 @@ def _run(self):
         self.frech_dists = self.frech_matrix[self.iu1]
 
     def tearDown(self):
+        try:
```

Comment:
should not be needed if you make sure to remove all references to `self.tmpdir`
```python
del self.tmpdir
```

It should auto-destruct.